integration-tests/go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/cespare/xxhash v1.1.0 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/chai2010/gettext-go v1.0.2 // indirect
-	github.com/chaos-mesh/chaos-mesh/api/v1alpha1 v0.0.0-20220226050744-799408773657 // indirect
+	github.com/chaos-mesh/chaos-mesh/api v0.0.0-20240709130330-9f4feec7553f // indirect
```

Comment:
I had to manually align this one with CTF to get the mod resolution to work.

Reply:
same question here: https://github.com/smartcontractkit/chainlink-starknet/pull/498#discussion_r1700540916